README.md

```diff
@@ -1,6 +1,15 @@
+<p>
+  <a href="https://ngrok.com">
+    <img src="assets/ngrok.png?raw=true" alt="ngrok Logo" width="300" url="https://ngrok.com" />
+  </a>
+</p>
```

Comment:
do we need this? suggest we remove this because it pushes more content 'below the fold'

Reply:
Ah I see. I was following the lead from the [k8s ingress controller](https://github.com/ngrok/kubernetes-ingress-controller), I assumed this is how we wanted the branding to come across. I can remove it.